src/test/kotlin/com/terraformation/backend/db/DatabaseTest.kt

```diff
@@ -949,12 +949,13 @@ abstract class DatabaseTest {
       organizationId: Any = this.organizationId,
       role: Role = Role.Contributor,
       createdBy: UserId = currentUser().userId,
+      createdTime: Instant = Instant.EPOCH,
   ) {
     with(ORGANIZATION_USERS) {
       dslContext
           .insertInto(ORGANIZATION_USERS)
           .set(CREATED_BY, createdBy)
-          .set(CREATED_TIME, Instant.EPOCH)
+          .set(CREATED_TIME, createdTime)
           .set(MODIFIED_BY, createdBy)
           .set(MODIFIED_TIME, Instant.EPOCH)
```

Comment:
Should set modified time to the created time too.

Reply:
Oops, yes good catch.